CSharp.lua/CoreSystem.Lua/CoreSystem/Reflection/Assembly.lua

```diff
@@ -1078,17 +1078,36 @@ define("System.Activator", {
 })
 
 define("System.Reflection.CustomAttributeExtensions", {
-  GetCustomAttribute = function (element, attributeType, inherit)
+  GetCustomAttributes = function (element, attributeType, inherit)
     if element == nil then throw(ArgumentNullException("element")) end
-    if attributeType == nil then throw(ArgumentNullException("attributeType")) end
     if type(attributeType) == "boolean" then
-      attributeType, inherit = inherit, attributeType
+      attributeType, inherit = nil, attributeType
+    elseif inherit == nil then
+      inherit = true
+    end
+    if attributeType == nil then
+      return element:GetCustomAttributes(inherit)
```

Comment:
To clarify this one, we call it like this to avoid getting an exception due to null attributeType.